src/focus-ring.js

```diff
@@ -74,11 +80,13 @@ document.addEventListener('DOMContentLoaded', function() {
    * opening a menu or dialog.
    */
   function onKeyDown() {
-    hadKeyboardEvent = true;
     // `activeElement` defaults to document.body if nothing focused,
     // so check the active element is actually focused.
-    if (matches(document.activeElement, ':focus'))
-      addFocusRingClass(document.activeElement);
+    var activeElement = document.activeElement;
+    if (activeElement.tagName.toLowerCase() == 'body')
```

Comment:
I am sure.

```html
<body></body><script>document.write(document.body.tagName)</script>
```

![Screenshot of Internet Explorer 8 displaying the tagName of body in uppercase](https://user-images.githubusercontent.com/188426/26841176-b475ec22-4ab6-11e7-803d-43c62fc00138.png)

In older IE, there would be an issue with new “unknown” elements (back then, HTML5 elements) returning lowercase. It’s unneeded here, because you’re testing for some of the oldest elements known to web. :)